ase/io/espresso_cp.py

```diff
@@ -76,9 +76,44 @@
 def write_espresso_cp_in(fd, atoms, input_data=None, pseudopotentials=None,
                       kspacing=None, kpts=None, koffset=(0, 0, 0),
                       **kwargs):
+
     write_espresso_in(fd, atoms, input_data, pseudopotentials,
```

Comment:
When we use `espresso_write_in` here, if I remove `NKSIC` and `EE` from `ase.io.espresso.KEYS` then this call will not write those blocks, so we need to do this manually